src/govuk/components/character-count/character-count.mjs

```diff
@@ -85,35 +78,12 @@ CharacterCount.prototype.init = function () {
   this.updateCountMessage()
 }
 
-// Read data attributes
-CharacterCount.prototype.getDataset = function (element) {
-  var dataset = {}
-  var attributes = element.attributes
-  if (attributes) {
-    for (var i = 0; i < attributes.length; i++) {
-      var attribute = attributes[i]
-      var match = attribute.name.match(/^data-(.+)/)
-      if (match) {
-        dataset[match[1]] = attribute.value
-      }
-    }
-  }
-  return dataset
-}
-
-// Counts characters or words in text
-CharacterCount.prototype.count = function (text) {
-  var length
-  if (this.options.maxwords) {
-    var tokens = text.match(/\S+/g) || [] // Matches consecutive non-whitespace chars
-    length = tokens.length
-  } else {
-    length = text.length
-  }
-  return length
-}
-
-// Bind input propertychange to the elements and update based on the change
+/**
+ * Bind change events
+ *
+ * Set up events on the $textarea so that the counts update when the user
```

Comment:
This makes it sound like internal counts are updating I think.

"... count messages update..."?